causalml/inference/meta/rlearner.py

```diff
@@ -267,7 +297,7 @@ def bootstrap(self, X, p, treatment, y, size=10000):
         p_b = {group: _p[idxs] for group, _p in p.items()}
```

Comment:
Let's change the order of input arguments in `bootstrap()` to be the same as `fit()`. i.e. `bootstrap(self, X, treatment, y, p, size)`